core-aam/aria-valuenow-spinbutton-default-manual.html

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<html>
+  <head>
+    <title>aria-valuenow</title>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" href="/wai-aria/scripts/manual.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/wai-aria/scripts/ATTAcomm.js"></script>
+    <script>
+    setup({explicit_timeout: true, explicit_done: true });
+
+    var theTest = new ATTAcomm(
+    {
+   "steps" : [
+      {
+         "element" : "test",
+         "test" : {
+            "ATK" : [
+               [
+                  "result",
+                  "atk_value_get_current_value()",
+                  "is",
+                  "0"
+               ]
+            ],
+            "AXAPI" : [
+               [
+                  "property",
+                  "AXValue",
+                  "is",
+                  ""
+               ]
+            ],
+            "IAccessible2" : [
+               [
+                  "property",
+                  "currentValue",
+                  "is",
+                  "VT_EMPTY"
+               ],
+               [
+                  "property",
+                  "accValue",
+                  "is",
+                  ""
```

Comment:
This is a string representation of the value, so can be empty string: https://docs.microsoft.com/en-us/windows/win32/api/oleacc/nf-oleacc-iaccessible-get_accvalue